scripts/fetch_and_build_pgxn

```diff
@@ -44,7 +44,6 @@ source /buildfiles/pkgvars
 # set default values for certain packaging variables
 declare pkglatest # to make shellcheck happy
 hubproj="${hubproj:-${pkgname}}"
-nightlypg="${nightlypg:-${releasepg}}"
```

Comment:
The current title of the PR and the change here seems unrelated. Can you elaborate?

You can consider moving this to another PR or updating the PR title, and description

Reply:
Thanks for your comments. I changed the title and description. Could you check it again?

Reply:
title remains unchanged
description still does not cover why `nightlypg` was removed from the files here.

Reply:
Apperantly, I didn't save the header :(. I changed it again.
Sorry for the loss of header